### PR TITLE
Update check for "No Results" when searching by name

### PR DESCRIPTION
**Pull request by CBXZero**
This should fix issue #93 that I opened. I check now for the div that encapsulates the full sized image (which is only found on the page when a single card is returned).

Also, sorry for the 2nd pull request. Messed up my first attempt at fixing it. Tested it extensively in my bot this time.

### CODE REVIEW

**Comment by CBXZero** [comment]
Just realized my mistake here, So the cardImage div won't work, looking to see if there's a better thing to use to fix this.

**Comment by davidchambers** [comment]
> So the cardImage div won't work, looking to see if there's a better thing to use to fix this.

Okay. Let me know if you discover a solution. :)

**Comment by CBXZero** [comment]
There we go, I got it figured out and I also wrote a test around it (haven't used coffeescript or mocha, so it took me a bit to figure out).

The issue ended up being that if you use a fun word like "test" as the name, then it would return multiple cards back (like here http://gatherer.wizards.com/Pages/Search/Default.aspx?name=+[test]). I decided to also check for the "Additional Sort:" text that appears on these pages to weed them out from providing false positives.

**Comment by CBXZero** [comment]
It look like my machine did some edits to the gatherer.js file when I was setting up my machine, do you want me to revert that file and then add in the changes that actually matter?

**Comment by davidchambers** [comment]
You should run `make setup` to install dependencies, then `make fixtures` to generate the fixtures, then `make test` to run the unit tests.

You should update the CoffeeScript files rather than the JavaScript files (see #92, for example). You shouldn't use your globally installed `coffee` to generate the JavaScript files – simply run `make`.

All this talk of Make assumes you're on OS X or some flavour of Linux. Is this the case?

**Comment by CBXZero** [comment]
Yeah, I'm running ubuntu of some variant on my laptop, I'll give that a shot, thanks :)

**Comment by CBXZero** [comment]
Fixed the coffeescript file and all the tests passed, and make setup made life about 10 times easier. Are there any other finer details I may be missing?

**Comment by davidchambers** [comment]
This is looking good. Please remove the __lib__ changes from the pull request. The generated files are automatically updated as part of the release process, so needn't be updated with each commit.

Also, please squash the commits. :)

File: test/tutor.coffee [hunk 1]
```diff
@@ -742,6 +742,11 @@ describe 'tutor.card', ->
       eq err.message, 'no results'
       eq card, undefined
 
+  it 'responds with "no results" given non-existent card name with multiple results',
```
Comment (davidchambers) [inline test/tutor.coffee]: Let's mention the relevant GitHub issue here, as we do on line 739.

File: src/gatherer.coffee [hunk 2]
```diff
@@ -32,7 +32,7 @@ gatherer.request = (args...) ->
       callback err
     else if res.statusCode isnt 200
       callback new Error 'unexpected status code'
-    else if body.indexOf('<title>Object moved</title>') >= 0
+    else if body.indexOf('<title>Object moved</title>') >= 0 && body.indexOf('Additional Sort') < 0
```
Comment (davidchambers) [inline src/gatherer.coffee]: Let's use `and` rather than `&&`, since we do so elsewhere.

**Comment by CBXZero** [comment]
somehow I just blew those up, let me fix whatever I have going on...

**Comment by CBXZero** [comment]
Somehow when I attempted to squash, I blew up everything in my branch and then some, I'm going to start fresh taking the notes that we have here and fixing this real quick. If I make another pull request for that I apologize in advance, first time contributing outside of team assignments at school.

**Comment by CBXZero** [comment]
Looks like it didn't want to update when I purged and redid this, So I'll have to move this into a new request, apologies for my Version Control mishap!

**Pull request closed without merging**